diff --git a/dataset_lossy.py b/dataset_lossy.py
index ff8be6d..de32e99 100644
--- a/dataset_lossy.py
+++ b/dataset_lossy.py
@@ -9,20 +9,20 @@ import random
 
 
 class Dataset(data.Dataset):
-    def __init__(self, root_dir, split, bit=10, maximum=20475, type='train', scaling_factor=1, time_step=1, format='npy'):
+    def __init__(self, root_dir, split, bit=10, maximum=20475, type='train', scaling_factor=1, time_step=1, format='ply'):
         self.maximum = maximum
         self.type = type
         self.scaling_factor = scaling_factor
         self.format = format
-        sequence_list = ['soldier', 'redandblack', 'loot', 'longdress', 'andrew', 'basketballplayer', 'dancer', 'david', 'exercise', 'phil', 'queen', 'ricardo', 'sarah', 'model']
+        sequence_list = ['soldier', 'redandblack', 'loot', 'longdress', 'andrew', 'basketball_player', 'dancer', 'david', 'exercise', 'phil', 'queen', 'ricardo', 'sarah', 'model']
         self.sequence_list = sequence_list
         start = [536, 1450, 1000, 1051, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1]
         end = [835, 1749, 1299, 1350, 317, 600, 600, 215, 600, 244, 249, 215, 206, 600]
         num = [end[i] - start[i] for i in range(len(start))]
         self.lookup = []
         for i in split:
-            sequence_dir = join(root_dir, sequence_list[i]+'_ori')
-            # sequence_dir = join(root_dir, sequence_list[i])
+            # sequence_dir = join(root_dir, sequence_list[i]+'_ori')
+            sequence_dir = join(root_dir, sequence_list[i])
             file_prefix = sequence_list[i]+'_vox'+str(bit)+'_'
             file_subfix = '.'+self.format
             if type == 'train':
diff --git a/dataset_owlii.py b/dataset_owlii.py
index d2228e8..178e240 100644
--- a/dataset_owlii.py
+++ b/dataset_owlii.py
@@ -9,12 +9,12 @@ import random
 
 
 class Dataset(data.Dataset):
-    def __init__(self, root_dir, split, bit=10, maximum=20475, type='train', scaling_factor=1, time_step=1, format='npy'):
+    def __init__(self, root_dir, split, bit=10, maximum=20475, type='train', scaling_factor=1, time_step=1, format='ply'):
         self.maximum = maximum
         self.type = type
         self.scaling_factor = scaling_factor
         self.format = format
-        sequence_list = ['basketball', 'dancer', 'exercise', 'model']
+        sequence_list = ['basketball_player', 'dancer', 'exercise', 'model']
         sequence_prefix = ['basketball_player', 'dancer', 'exercise', 'model']
         self.sequence_list = sequence_list
         start = [1, 1, 1, 1]
@@ -36,8 +36,8 @@ class Dataset(data.Dataset):
                 s = start[i]
                 e = end[i]
             for s in range(s, e):
-                s1 = str(s+time_step).zfill(8)
-                s0 = str(s).zfill(8)
+                s1 = str(s+time_step).zfill(4)
+                s0 = str(s).zfill(4)
                 file_name0 = file_prefix + s0 + file_subfix
                 file_name1 = file_prefix + s1 + file_subfix
                 file_dir = join(sequence_dir, file_name0)
diff --git a/test_owlii.py b/test_owlii.py
index 60d621d..88b1865 100644
--- a/test_owlii.py
+++ b/test_owlii.py
@@ -123,14 +123,14 @@ if __name__ == '__main__':
     lossless_model = lossless_model.to(device).eval()
 
     results = {
-        'basketball': {'bpp': [], 'd1-psnr': [], 'd2-psnr': [], 'cd': [], 'exp_name': []},
+        'basketball_player': {'bpp': [], 'd1-psnr': [], 'd2-psnr': [], 'cd': [], 'exp_name': []},
         'dancer': {'bpp': [], 'd1-psnr': [], 'd2-psnr': [], 'cd': [], 'exp_name': []},
         'exercise': {'bpp': [], 'd1-psnr': [], 'd2-psnr': [], 'cd': [], 'exp_name': []},
         'model': {'bpp': [], 'd1-psnr': [], 'd2-psnr': [], 'cd': [], 'exp_name': []}
     }
     '''
     start testing
-    5: basketballplayer
+    5: basketball_player
     6: dancer
     8: exercise
     13: model
